web/packages/teleport/src/TopBar/TopBar.tsx

```diff
@@ -84,9 +143,44 @@ export function TopBar() {
         onLoad={loadClusters}
         style={styles}
       />
-      <Flex ml="auto" height="100%">
+      <Flex ml="auto" height="100%" alignItems="center">
+        {!hasDockedElement && assistEnabled && (
+          <AssistButton onClick={() => setShowAssist(true)}>
+            <BrainIcon />
+          </AssistButton>
+        )}
         <UserMenuNav username={ctx.storeUser.state.username} />
       </Flex>
+
+      {showAssistPopup && (
```

Comment:
possible to align carrot center to the brain? (using chrome)
minor: can we also put a hover state to the `close` button?

<img width="391" alt="image" src="https://github.com/gravitational/teleport/assets/43280172/50a9bd3a-cc97-4335-8537-a6700bbe168d">

Reply:
Aha I had only aligned it to my username, forgetting that changes in length